bootstrap.php

```diff
@@ -223,6 +241,27 @@
 $GLOBALS['processedRecords'] = 0;
 $GLOBALS['allProcessedRecords'] = 0;
 
+/*
+ * if user wants all modules, append system modules to existing config
+ */
+if (isset($opts['allmodules'])) {
+    global $moduleList;
+    foreach ($moduleList as $aModule) {
+        if (!isset($modules[$aModule])) {
+            $modules[$aModule] = (is_numeric($opts['a']) ? $opts['a'] : 5000);
```

Comment:
I guess this is typo, ```allmodules``` should be used instead of ```$opts['a']``` - $opts['allmodules'].
Also, it will be great to put 5000 in config, so this value could be overriden in custom configs in future, without modifying the code.

I'm going to merge this PR https://github.com/sugarcrm/Tidbit/pull/50 today, so please update your branch.
Thanks!